torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1114,7 +1114,12 @@ void TensorExprKernel::lowerToBackend(BackendType backendType) {
 
   l.ApplyInlines();
   Stmt* stmt = l.root_stmt();
-  stmt = IRSimplifier::simplify(stmt);
+
+  try {
+    stmt = IRSimplifier::simplify(stmt);
+  } catch (const std::exception& e) {
+    std::cout << "Failed IR Simplification " << e.what() << "\n";
```

Comment:
We probably should handle it somehow differently than just complaining in stdout. Just ignore if simplification failed for now? Ultimately I think simplification should never fail - it could do nothing to the stmt, but it should never throw.

Reply:
There are some cases which throw due to malformed inputs, I think it's infeasible to completely ignore them, but we could move the catch into the call to simplify?

Reply:
Since simplify should only be called internally, we should guarantee the well-formed inputs for it. If we have a bug somewhere, we could throw an error, but it shouldn't be a job of simplify to catch it - it should be caught one scope higher I believe (indicating that the issue is not in simplify but rather in the place that produced the malformed inputs for simplify).